lib/toggle-markdown-task.js

```diff
@@ -40,7 +40,7 @@ function toggleSelection (selection) {
 }
 
 function toggleTask (taskText) {
-  const regex = new RegExp(/([\-\*]\ )(\[[\ x]\])/)
```

Comment:
Is this necessary? Doesn't seem related.

Reply:
@RichardLitt: Thanks for the review. :bow:

This change is indeed related, but I should have made that clear in the pull request description. This change resolves existing linter violations related to unnecessary escape characters. See 1b980196a6fea73ffd69150d82456b64916f0e05.

I've updated the pull request description to clarify that this pull request sets up the linter _and_ resolves existing linter violations.